.github/workflows/pr.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         export GOPATH=$HOME/go
         export PATH=$PATH:$GOPATH/bin
         make install-tools
-        kind create cluster --image kindest/node:v1.19.1
+        kind create cluster --image kindest/node:v1.19.4
```

Comment:
1.20 is GA, right? So I feel we should probably be testing on 1.20 by default wherever we have a default (like here)

Reply:
Good point and updated!